index.html

```diff
@@ -40,6 +40,222 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                    <div class="primarytext1">
+                        <div class="multi-post">
+    
+                            Multi-Post
+                            <br>
+                            Stories
+    
+                        </div>
+    
+                        <div class="fram">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <span>A daily selection of privately</span>
+                        <span>personalized reads;no accounts or</span>
+                        <span> sign-ups required.</span>
+                    </div>
+                    <div class="primarytext3">
+                        <button class="lang">HTML</button>
+                        <button class="lang">CSS</button>
+                        <button class="lang JavaScript">JavaScript</button>
+                    </div>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="about-my-slef">
+                <div class="top">
+                    <div class="first-text">About <br> Myself</div>
+                    <div class="second-text">About Myself
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through
+                        some of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hesitate to contact me.
+                        LET’S CONNECT
+                        Get my resume
+                        Languages
+                        JavaScript
+                        HTML
+                        CSS
+                        Frameworks
+                        Skills</div>
+    
+    
+                    <div class="social-media1">
+                        <div class="sub-social-media">
+                            <span>LET’S CONNECT</span>
+                        </div>
+                        <div class="socail-media-icon1">
+                            <a href="#"><img src="./Image/Vector (1).png" alt=""></a>
+                            <a href="#"><img src="./Image/Linkedin icon.png" alt=""></a>
+                            <a href="#"><img src="./Image/Vector (4).png" alt=""></a>
+                            <a href="#"><img src="./Image/Vector.png" alt=""></a>
+                            <a href="#"><img src="./Image/Vector (3).png" alt=""></a>
+    
+                        </div>
```

Comment:
- [ ] Kindly ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) because the `ul` tag better adds meaning to list of items in a website 👍🏽